### ISSUE TITLE
Publish Error of Blazor wasm, TFM=.net6+aot, .net 7 sdk installed

### ISSUE THREAD

**Issue by ahmed-salem-me** [dotnet/runtime#69731]
### Description

Publish a blazor wasm fails with the following errors:
```
Search paths being used for $(MSBuildExtensionsPath) are C:\Apps\Visual Studio\VS 2022 Community\MSBuild;$(MSBuildProgramFiles32)\MSBuild
Trying to import C:\Apps\Visual Studio\VS 2022 Community\MSBuild\Microsoft\VisualStudio\v$(VisualStudioVersion)\TypeScript\Microsoft.TypeScript.Default.props using extensions path C:\Apps\Visual Studio\VS 2022 Community\MSBuild
The "TargetFramework" property is a global property, and cannot be modified.
The "Configuration" property is a global property, and cannot be modified.
The "Platform" property is a global property, and cannot be modified.
The "WebPublishProfileFile" property is a global property, and cannot be modified.
The "TargetFramework" property is a global property, and cannot be modified.
Property reassignment: $(TargetPlatformVersion)="" (previous value: "0.0") at C:\Program Files\dotnet\sdk\7.0.100-preview.4.22252.9\Sdks\Microsoft.NET.Sdk\targets\Microsoft.NET.TargetFrameworkInference.targets (65,5)
Property reassignment: $(DefaultItemExcludes)=";**\node_modules\**;node_modules\**;**\jspm_packages\**;jspm_packages\**;**\bower_components\**;bower_components\**;bin\Release\/**" (previous value: ";**\node_modules\**;node_modules\**;**\jspm_packages\**;jspm_packages\**;**\bower_components\**;bower_components\**") at C:\Program Files\dotnet\sdk\7.0.100-preview.4.22252.9\Sdks\Microsoft.NET.Sdk\targets\Microsoft.NET.TargetFrameworkInference.targets (225,5)
Property reassignment: $(DefaultItemExcludes)=";**\node_modules\**;node_modules\**;**\jspm_packages\**;jspm_packages\**;**\bower_components\**;bower_components\**;bin\Release\/**;obj\Release\/**" (previous value: ";**\node_modules\**;node_modules\**;**\jspm_packages\**;jspm_packages\**;**\bower_components\**;bower_components\**;bin\Release\/**") at C:\Program Files\dotnet\sdk\7.0.100-preview.4.22252.9\Sdks\Microsoft.NET.Sdk\targets\Microsoft.NET.TargetFrameworkInference.targets (226,5)
Property reassignment: $(IntermediateOutputPath)="obj\Release\net6.0\" (previous value: "obj\Release\") at C:\Program Files\dotnet\sdk\7.0.100-preview.4.22252.9\Sdks\Microsoft.NET.Sdk\targets\Microsoft.NET.TargetFrameworkInference.targets (244,5)
Property reassignment: $(OutputPath)="bin\Release\net6.0\" (previous value: "bin\Release\") at C:\Program Files\dotnet\sdk\7.0.100-preview.4.22252.9\Sdks\Microsoft.NET.Sdk\targets\Microsoft.NET.TargetFrameworkInference.targets (245,5)
Resolving SDK 'Microsoft.NET.Runtime.Emscripten.Python'...
Project "BlazorApp3.csproj" (build target(s)):
C:\Program Files\dotnet\sdk-manifests\7.0.100\microsoft.net.workload.emscripten\WorkloadManifest.targets(22,5): Error MSB4019: The imported project "C:\Program Files\dotnet\packs\Microsoft.NET.Runtime.Emscripten.3.1.7.Python.win-x64\7.0.0-preview.4.22215.1\Sdk\Sdk.props" was not found. Confirm that the expression in the Import declaration "C:\Program Files\dotnet\packs\Microsoft.NET.Runtime.Emscripten.3.1.7.Python.win-x64\7.0.0-preview.4.22215.1\Sdk\Sdk.props" is correct, and that the file exists on disk.
Done building project "BlazorApp3.csproj" -- FAILED.
```
arch-wasm
area-Codegen-AOT-mono

### Reproduction Steps

Have the following sdks installed: .net6.0.3 , 7.0.100-preview.4.22252.9
remove whatever current blazor wasm-tools -> dotnet workload uninstall wasm-tools
Create a global.json with sdk:6.0.3
install wasm-tools , it installs according to version 6.0.3
Create a new blazor wasm project, self hosted, TFM=.net6
add global.json with sdk:6.0.3 to project dir.
build release . builds successfully 
publish, give the above mentioned errors.

### Expected behavior

publish successfully 

### Actual behavior
`
Error MSB4019: The imported project "C:\Program Files\dotnet\packs\Microsoft.NET.Runtime.Emscripten.3.1.7.Python.win-x64\7.0.0-preview.4.22215.1\Sdk\Sdk.props" was not found
`

**Comment by radical** [comment]
@ahmed-salem-me this is not supported yet. And we are tracking the issue in https://github.com/dotnet/runtime/issues/65211, and you can watch that for the fix.

Closing as a duplicate of https://github.com/dotnet/runtime/issues/65211 .

**Comment by ahmed-salem-me** [comment]
> @ahmed-salem-me this is not supported yet. And we are tracking the issue in #65211, and you can watch that for the fix.
> 
> Closing as a duplicate of #65211 .

@radical , Just to be sure this is an exact similar case as the linked issue, can you please look at the "reproduction steps" in this issue.
Cause here I used the global.json file for two things, first to get the wasm-tools install the 6.0.3 version of it and to have the primary/active sdk set to .net6 for the project I'm publishing
So theatrically here .net 7 sdk is not/should not be used to publish a .net6 app. !

**Comment by radical** [comment]
Can you share the binlog? Add `-bl` to the command line, and attach the generated `msbuild.binlog`.

**Comment by ahmed-salem-me** [comment]
> Can you share the binlog? Add `-bl` to the command line, and attach the generated `msbuild.binlog`.

@radical , Okay so I've moved the global.json to solution folder instead of project folder and it published successfully.

So I can say that my approach works, and can allow you to publish in .net6 aot , while .net7 sdk is installed.
I've tried a new blazor app, then tried my original big project. both published.
You do have to uninstall wasm-tools and re-install it with .net6 version though.
